python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -42,12 +50,13 @@ def get_dimensions(dataset):
     return dimensions, types
 
 
-def get_partitions(dku_flow_variables, dimensions):
+def get_partitions(dku_flow_variables, dimensions, input_id=None):
```

Comment:
Dummy question: Is there any way we could remove the need to specify the input_id?

Reply:
@alexbourret ?

Reply:
After investigation, I understand this is mandatory to get partition values from an input folder since the necessary environment variable is DKU_SRC_FOLDER_ID_XXXX. Closing this.